config/locales/statuses/en.yml

```diff
@@ -143,10 +143,10 @@ en:
           be paid until they’ve specified a valid payment URL.
       accepted:
         header: "Accepted"
-        body: "%{winner_email}'s delivered code was accepted on %{accepted_at}."
+        body: "%{winner_url}'s delivered code was accepted on %{accepted_at}."
       rejected:
         header: "Rejected"
-        body: "<a href=%{delivery_url}>%{winner_email}'s work</a> was rejected on %{rejected_at}."
+        body: "<a href=%{delivery_url}>%{winner_name}'s work</a> was rejected on %{rejected_at}."
```

Comment:
Oh, I guess that's why. I wonder if we should have winner_url and their work as two separate links but I guess that is a bit confusing